### PR TITLE
Add CrackProof packer rule

### PR DESCRIPTION
**Pull request by enovella**
Fix https://github.com/rednaga/APKiD/issues/139

### CODE REVIEW

File: apkid/rules/elf/packers.yara [hunk 1]
```diff
@@ -371,3 +371,80 @@ rule tencent_elf : packer
     is_elf
     and any of them
 }
+
+rule crackproof : packer
+{
+	meta:
+		description = "CrackProof"
+    url         = "https://www.hypertech.co.jp/eng/"
```
Comment (strazzere) [inline apkid/rules/elf/packers.yara]: Tab mismatching

File: apkid/rules/elf/packers.yara [hunk 2]
```diff
@@ -371,3 +371,80 @@ rule tencent_elf : packer
...
+		*/
+		$do_asm_syscall = {
+			FE 4F 2D E9  //  PUSH  {R1-R11,LR}
+			2C B0 8D E2  //  ADD   R11, SP, #0x2C
+			04 40 9B E5  //  LDR   R4, [R11,#a5]
+			08 50 9B E5  //  LDR   R5, [R11,#a6]
+			0C 60 9B E5  //  LDR   R6, [R11,#a7]
+			10 70 9B E5  //  LDR   R7, [R11,#svc_nr]
+			00 00 00 EF  //  SVC   0
+			FE 8F BD E8  //  POP   {R1-R11,PC}
```
Comment (strazzere) [inline apkid/rules/elf/packers.yara]: Is this specific to this packer? Seems like a compile/library optimization?

Comment (enovella) [reply apkid/rules/elf/packers.yara]: yes it is what I used a while ago.

File: apkid/rules/elf/packers.yara (hunk 1 above)
Comment (enovella) [reply apkid/rules/elf/packers.yara]: Fixed in the other commits....

**Comment by enovella** [comment]
> Are we positive that this isn't a compiler/library optimization that we're pinning against?
> 
> I see it's hitting multiple apks, are the shared libs different hashes?

Sure about it..... They have Java, native and unity protection. Take a look at the issue to verify its correctness, or even better grab any of the samples and disassemble them. ;-)

**Comment by strazzere** [comment]
Actually, this is a really good example you've provided.

```
[+] APKiD 2.1.0 :: from RedNaga :: rednaga.io
[*] ./libmgRun_02.10.26_01.so.zip!libmgRun_02.10.26_01.so
 |-> packer : UPX (unknown, modified)
```

The reason it wouldn't match is because that lib has been packed using UPX, slightly munged on top of the normal obfuscation.

**Comment by enovella** [comment]
> Actually, this is a really good example you've provided.
> 
> ```
> [+] APKiD 2.1.0 :: from RedNaga :: rednaga.io
> [*] ./libmgRun_02.10.26_01.so.zip!libmgRun_02.10.26_01.so
>  |-> packer : UPX (unknown, modified)
> ```
> 
> The reason it wouldn't match is because that lib has been packed using UPX, slightly munged on top of the normal obfuscation.

you have mixed the post. This is not in here.

**Comment by strazzere** [comment]
Works for me, I don't have time to dig into them all as of right now

**Pull request merged**